docs/workflow.rst

```diff
@@ -396,12 +297,12 @@ Create the merged stats file
 ============================
 
 The stats (catalog of fit parameters) files can then be merged into a single
-file for the region.  This only merges the stats output files, but not the
+file for the field.  This only merges the stats output files, but not the
 pdf1d or lnp files (see the next section).
 
   .. code-block:: console
 
-     $ beast/tools/merge_stats_file.py filebase
+     $ beast.tools.run.merge_files --use_sd 1
```

Comment:
Need to covert commands to command line versions (see prev comments).

Reply:
resolved per discussion below